### PR TITLE
fixes #175 - Created tutorial for creating loklak apps

### LINKED ISSUE #175: Create a tutorial for creating loklak apps
Create a tutorial page which will contain detailed information about creating loklakapps. Detailed information about configuring the app.json should be mentioned here. Sample app.json should be provied.

### PR DESCRIPTION
**Pull request by djmgit**
fixes issue #175. Created a detailed describing how to
create a loklak app. Updated the app.json of boilerplate
app and prepared it for store listing as it has been referred
in the tutorial.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
@kavithaenair, @SKrPl, @Achint08 open for review

File: docs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+```
+git push origin master branch_name
+```
+
+Provide your credentials and changes will be pushed into a new branch in your forked repo
+at github. Now simply head to your forked apps.loklak.org repository and select the newly
+created branch and send a pull request.
+
+Soon your app will be reviwed by your co-developers at FOSSASIA, if required you may be asked to
```
Comment (kavithaenair) [inline docs/tutorial.md]: Typo here

Comment (kavithaenair) [inline docs/tutorial.md]: `asked to changes` ? Sentence is not right i guess.

File: docs/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+Enter apps.loklak.org repo and execute the below commands.
+
+```
+git add --all
+git commit
+```
+This stages your newly added app for commit, and tries to commit the changes.
+Provide a suitable commit message and do not forget to refer the issue you just created.
+Once commit is success full, you are ready to push the changes.
```
Comment (kavithaenair) [inline docs/tutorial.md]: `success full` ?

File: docs/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    "email":"djmdeveloper060796@gmail.com",
+    "url":"https://github.com/djmgit",
+    "sameAs":"https://github.com/djmgit"
+  }
+}
+```
+
+Here some of the mose important fields which must be set are :-
+
+**name** : It is the name of yout application.
```
Comment (kavithaenair) [inline docs/tutorial.md]: Typo here

File: docs/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    "@type":"Person",
+    "name":"Deepjyoti Mondal",
+    "email":"djmdeveloper060796@gmail.com",
+    "url":"https://github.com/djmgit",
+    "sameAs":"https://github.com/djmgit"
+  }
+}
+```
+
+Here some of the mose important fields which must be set are :-
```
Comment (kavithaenair) [inline docs/tutorial.md]: Typo here

File: docs/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+After successfully creating your loklak application you need to document your app so
+that it can be properly showcased on apps.loklak.org's storelisting page and others
+can know and understand your app before actually using them.
+
+If you have used Boilerplate or loklakinit to create your app, then you will already
+have three .md files in your app directory, or else you need to create those yourself.
+These are getStarted.md, this should contain all necessary information regarding getting
+started with your app, how to use your app, what your app does, what does the various
+fields inside your app mean, etc. User should get a complete understanding of your app
+form this.
```
Comment (kavithaenair) [inline docs/tutorial.md]: Typo

**Comment by kavithaenair** [comment]
@djmgit Please check the comments.

**Comment by djmgit** [comment]
@kavithaenair did the changes

**Comment by djmgit** [comment]
@mariobehling removed the changes you mentioned

**Comment by mariobehling** [comment]
Thanks!

**Pull request merged**